--- a/recipes/gst-plugins-bad-1.0.recipe
+++ b/recipes/gst-plugins-bad-1.0.recipe
@@ -14,42 +14,42 @@
     # we error out if they weren't found.
     meson_options = {'gpl': 'disabled',
                      # enabled plugins
-                     'aes': 'enabled',
-                     'analyticsoverlay': 'enabled',
-                     'assrender': 'enabled',
-                     'bz2': 'enabled',
-                     'closedcaption': 'enabled',
-                     'codecalpha': 'enabled',
-                     'codectimestamper': 'enabled',
-                     'dash': 'enabled',
-                     'decklink': 'enabled',
-                     'dtls': 'enabled',
-                     'dts': 'enabled',
+                      'aes': 'disabled',
+                      'analyticsoverlay': 'disabled',
+                      'assrender': 'disabled',
+                      'bz2': 'disabled',
+                      'closedcaption': 'disabled',
+                      'codecalpha': 'disabled',
+                      'codectimestamper': 'disabled',
+                      'dash': 'disabled',
+                      'decklink': 'disabled',
+                      'dtls': 'disabled',
+                      'dts': 'disabled',
                      'hls': 'enabled',
                      'hls-crypto': 'openssl',
-                     'ivfparse': 'enabled',
-                     'ladspa': 'auto', # lrdf is optional
-                     'lcevcdecoder': 'enabled',
-                     'openh264': 'enabled',
-                     'openjpeg': 'enabled',
-                     'qroverlay': 'enabled',
-                     'opus': 'enabled',
-                     'rtmp': 'enabled',
-                     'rtmp2': 'enabled',
-                     'sbc': 'enabled',
-                     'sctp': 'enabled',
-                     'sctp-internal-usrsctp': 'enabled',
-                     'soundtouch': 'enabled',
-                     'srt': 'enabled',
-                     'spandsp': 'enabled',
-                     'srtp': 'enabled',
-                     'resindvd': 'enabled',
-                     'rist' : 'enabled',
-                     'voaacenc': 'enabled',
-                     'vulkan': 'enabled',
-                     'webrtc': 'enabled',
-                     'webrtcdsp': 'enabled',
-                     'zbar': 'enabled',
+                      'ivfparse': 'disabled',
+                      'ladspa': 'disabled',
+                      'lcevcdecoder': 'disabled',
+                      'openh264': 'disabled',
+                      'openjpeg': 'disabled',
+                      'qroverlay': 'disabled',
+                      'opus': 'disabled',
+                      'rtmp': 'disabled',
+                      'rtmp2': 'disabled',
+                      'sbc': 'disabled',
+                      'sctp': 'disabled',
+                      'sctp-internal-usrsctp': 'disabled',
+                      'soundtouch': 'disabled',
+                      'srt': 'disabled',
+                      'spandsp': 'disabled',
+                      'srtp': 'disabled',
+                      'resindvd': 'disabled',
+                      'rist' : 'disabled',
+                      'voaacenc': 'disabled',
+                      'vulkan': 'disabled',
+                      'webrtc': 'disabled',
+                      'webrtcdsp': 'disabled',
+                      'zbar': 'disabled',
                      # disabled plugins
                      'amfcodec': 'disabled',
                      'asio': 'disabled',
@@ -85,12 +85,8 @@
                      'win32ipc': 'disabled',
                      'wildmidi': 'disabled',
                     }
-    deps = ['gstreamer-1.0', 'gst-plugins-base-1.0', 'bzip2', 'libass',
-            'zlib', 'openh264', 'opus', 'lcevcdec', 'librtmp', 'libxml2',
-            'libsrtp', 'libdca', 'libdvdnav', 'libnice',
-            'soundtouch', 'vo-aacenc', 'openjpeg', 'openssl',
-            'pango', 'spandsp', 'webrtc-audio-processing', 'sbc', 'ladspa',
-            'srt', 'zbar', 'libltc', 'qrencode', 'json-glib']
+    deps = ['gstreamer-1.0', 'gst-plugins-base-1.0', 'zlib', 'libxml2',
+            'openssl', 'libva']
     platform_deps = {
         Platform.LINUX: ['libdrm']
     }
@@ -532,12 +528,8 @@
     }
 
     def prepare(self):
-        # Enable hardware codecs if variants are enabled since they have
-        # external dependencies such as SDKs.
-        self.enable_plugin('msdk', 'codecs', 'intelmsdk')
         self.enable_plugin('va', 'codecs', 'va', dep='libva')
 
-        self.enable_plugin('rsvg', 'codecs', variant='rust', dep='librsvg')
 
         if self.config.target_subsystem == Subsystem.TVOS_SIMULATOR and self.config.target_arch == Architecture.X86_64:
             # svtav1 plugin fails to link with the static svt-av1 lib on tvOS
@@ -608,20 +600,6 @@
             self.enable_plugin('mediafoundation', 'sys')
             self.enable_plugin('wasapi2', 'sys')
 
-        if self.config.target_platform in (Platform.LINUX, Platform.WINDOWS):
-            self.enable_plugin('nvcodec', 'codecs')
-            self.files_plugins_devel += [
-                'include/gstreamer-1.0/gst/cuda',
-                '%(libdir)s/pkgconfig/gstreamer-cuda-1.0.pc',
-            ]
-            self.files_typelibs += [
-                'GstHip-1.0',
-                'GstHipGL-1.0',
-            ]
-
-        if not Platform.is_app_platform(self.config.target_platform):
-            self.enable_plugin('curl', 'net', dep='curl')
-
         if self.config.target_platform not in (Platform.ANDROID, Platform.WINDOWS):
             self.meson_options['shm'] = 'enabled'
 
@@ -650,10 +628,6 @@
         if self.config.variants.nodebug:
             self.append_env('CFLAGS', '-DGST_LEVEL_MAX=GST_LEVEL_FIXME')
 
-        if self.config.target_platform in (Platform.WINDOWS, Platform.LINUX):
-            if self.config.target_arch == Architecture.X86_64:
-                self.enable_plugin('svtjpegxs', 'codecs', dep='svt-jpeg-xs')
-
         if self.config.target_platform == Platform.WINDOWS:
             if self.config.target_arch != Architecture.X86_64:
                 self.disable_plugin('lcevcdecoder', 'codecs', dep='lcevcdec')
@@ -662,9 +636,6 @@
             # Needs libiconv, which uses Autotools
             self.disable_plugin('zbar', 'codecs', dep='zbar')
 
-        if self.config.target_platform != Platform.WINDOWS:
-            self.enable_plugin('unixfd', 'effects')
-
     def post_install(self):
         LibtoolLibrary('gsturidownloader-1.0', None, None, None, self.config.libdir,
                 self.config.target_platform, deps=['gstreamer-1.0', 'gstbase-1.0']).save()
